sqlitedict.py

```diff
@@ -300,6 +299,12 @@ def clear(self):
         self.conn.execute(CLEAR_ALL)
         self.conn.commit()
 
+    def get_tablenames(self):
+        """print tablenames and return them as list"""
+        res = self.conn.select("SELECT name FROM sqlite_master WHERE type='table';")
```

Comment:
-1 on code duplication, it makes maintenance a headache. Keep it [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Can you refactor the core functionality to live in only one place? Possibly being used/called from multiple places, that's fine.